Source/Dafny/Compilers/Compiler-java.cs

```diff
@@ -16,6 +16,7 @@
 using System.Diagnostics;
 using System.Text.RegularExpressions;
 using System.Reflection;
+using Microsoft.Win32.SafeHandles;
```

Comment:
Is this used somehow? Hopefully not?

Reply:
Ah, IDEs! I was exploring various APIs to use to emit `DafnyRuntime.jar`, and my IDE must have inserted that while I had something temporary in place that I removed as soon as I read the docstring. :)